.github/dependabot.yml

```diff
@@ -8,10 +8,10 @@ updates:
   - package-ecosystem: gomod
     directory: "/"
     schedule:
-      interval: "weekly"
+      interval: "monthly"
```

Comment:
what's the motivation for moving this from `weekly` -> `monthly`?

Reply:
we just depend on upstream for types. Not as high a priority vs standard chains depending. Easier to maintainer / less PR spam